src/main/java/seedu/application/model/job/Industry.java

```diff
@@ -9,6 +9,7 @@
  */
 
 public class Industry {
+    public static final String INDUSTRY_SPECIFIER = "-i";
```

Comment:
Hmm do you think it will be better to place all the Specifiers in the `CliSyntax` class? Or in another class by itself?

In similar sense, do you think it will be a good idea to implement another class like `Specifier.java` to handle the operations w field specifiers just like `Prefix.java`